README.md

```diff
@@ -1,4 +1,5 @@
+mario_23 está desconectado  	
```

Comment:
hola

Reply:
Gaming

Reply:
AisakBrown
/
hello-world2
Code
Issues
one
Pull requests
two
Actions
Projects
Wiki
Security
Insights
Conversation	Commits	Changes
Jump to bottom Open
Update README.md # 3
hek189 : patch-2

@ hek189
hek189 opened this pull request
about 1 year ago
No description given.
hek189 added a commit about 1 year ago
@ hek189
Update README.md
@PedroCimadevilla
PedroCimadevilla
commented 8 months ago
Hello

@PedroCimadevilla
PedroCimadevilla
commented 8 months ago
public int listar(File origen) throws IOException { if (origen.isDirectory()) { String ficheros[] = origen.list(); for (String fichero : ficheros) { File ficheroOrigen = new File(origen, fichero); contabilidad++; listar(ficheroOrigen); } } else { contabilidad++; } return contabilidad; }

@PedroCimadevilla	PedroCimadevilla reviewed 8 months ago
to

README.md
@@ -1.4 +1.5 @@
# hello-world2
hello-world2
isaac piece of shit
signed: isaac
Isaac Hueles quite regular honestly
I shit on all the dead of your genealogical tree and if you hurry me also on the living, you fucking fucking amorous I catch you in the street and I sink your pexo with hammer blows, you sick bitch, if you have children I hope they have some physical disability or mental or failing that a bus runs over them, but do not die they suffer all their fucking life and if you have no children it will never be what will happen surely God bless you with a fat bitch fucking motherfucker, as I found out that you add another Once with an account like that and I found out who you are, I swear I am looking for you,When I find you I will take parts of your ridiculous body and I will eat them and while I eat them I will shit them and I will make you eat my fucking feces with pieces of your skin battered and when I have completely gutted you and made you eat all the shit to let go of my precious and shiny ass I will go for your sister and if you do not have a sister I will go for the sweat of your fucking mother and if I am inspired I will go for both of them, I will kidnap her, I will put them in a van, I will take them to a room, I will put the tail through all the holes in his body (yes,including those of the nose and ears) runs inside them and I will wait 9 months for their daughters to be born and when they turn 13 I will fuck them too and if even so after that you still have cousins ​​or aunts I will do the same with them and When I'm tired of fucking all your family of lice, I will limp a few chains I will put them in my car and I will travel 300km with all your family hooked on them and if after that someone is left alive,I made him alcohol to rage even more pain and after all that I will go to the hospital when you have recovered from the gut I made you I will take you out of there I will take you to the same room where I fucked all the women in your current family and to those that precede your family tree and while I put the videos of how I fucked your mother I will give you mini jaws on the forehead until you get the mark of my great and devastating glans for the rest of your life and so every time that you look in the mirror you will remember those videos and what I did with your family, after that I will release you and I will come back for you after three months, I will put you back in the room, but this time nothing smooth,This time I will limp your hands and I will start putting needles between your nails until the level of pain makes you pass out and I will revive you with a defibrillator, I will lower your pants and underpants and I will start giving you mini-jabs on your balls until little by little They are undoing and your scrotum is completely empty, I imagine that after that you would pass out again, because I will use the defibrillator again to revive you and put your feet in a bucket of water, I will put nippers, penis and tongue clamps and I will give you You download until you pass out again, when you do you already know what I will do ... and I will come back and limp some pliers and I will tear off your fucking nails piece of scum, then I will knock you down, I will put a rag on your face and I will make you little by little in your mouth without drowning ...Then I will leave and come back every day to make you a different torture, so that every time you hear my steps approach the door at different times each day, a fear that you have never experienced runs through your body and stay at the door pretending to speak. until you piss on me, then I will come in and start ... when I see that you no longer give for more torture I will let you heal in a hospital and I will come back for you when you recover, I limp and while I burn your eyes with a blowtorch will hammer you in the walnut until you die piece of son of a great bitch but do not think that everything ends up there ... if reincarnation exists,I will be reincarnated in any other person and I will do to you and all your new family everything I have done to you in this life but slower and without killing you so that every time you see a shadow at night you think that it is me, that a impressive madness, that you fall into a vegetative state simply from the fear that my presence causes you, if you are cured, I will not come to you anymore, you will die old, alone, without eggs, without skin and hope that you do not cut your legs waste shitty
mario_23 is offline  	
@PedroCimadevilla  PedroCimadevilla • 8 months ago
Hello

@ hek189  hek189 • 7 months ago
Gaming

Comment on pull request
Leave a comment
Notifications for this thread
You're not receiving notifications from this threat